src/main/resources/application.yml

```diff
@@ -18,7 +18,9 @@ decrypt:
     targetContainer:
       rtd: rtd-transactions-decrypted
       ade: ade-transactions-decrypted
-  
+
+splitter:
```

Comment:
@TommasoLencioni Maybe we could move this property under decrypter

Reply:
Thanks, it is going to be moved in the next splitter refinement.